docs/docs/twig/ref.md

```diff
@@ -2,13 +2,13 @@
 
 The `ref` attribute can be used to export an element so that a component can gain access to it.
 
-The `ref` attribute must **not** be filtered through the `attrs` filter.
-
 The `ref` attribute is not visible on the rendered HTML.
 
+It is recommended that the `ref` attribute is filtered via the `attrs` filter so that it is not exposed to the HTML.
+
 ## Example
 ```html
-<button ref="arrow" type="button" class="btn--reset">
+<button {{ { ref: "arrow" } | attrs }} type="button" class="btn--reset">
```

Comment:
That approval was a bit too early, sorry.

Ideally, the value of `ref` should no longer be a string but instead a function (accepts the element and returns an object with an `unsubscribe` method). That helps significantly with maintaining the references, eases cleanup (to avoid memory leaks) and is much nicer when the referred HTML element changes throughout the lifetime of the component.